src/selectorNameBase.ts

```diff
@@ -69,7 +69,7 @@ class SelectorNameValidatorWalker extends Lint.RuleWalker {
   }
 
   private validateSelector(className: string, arg: ts.Node) {
-    if (arg.kind === SyntaxKind.current().ObjectLiteralExpression) {
+    if (arg && arg.kind === SyntaxKind.current().ObjectLiteralExpression) {
```

Comment:
@preslavsh would this also not mean that the selectors are not going to be validated anymore?